### DISCUSSION TITLE
Shouldn't NavContainer inside Navbar overrides the existing container?

### DISCUSSION THREAD

**Discussion by munaja** [themesberg/flowbite-svelte#1364] [Q&A]
As stated in the [Documantation here](https://flowbite-svelte.com/docs/components/navbar#Navbar_container) , NavContainer should gives more control for the container. But when I add one, it creates new container instead of replacing the existing one. Resulting in I can't utilize the fluid feature. or..
is there a way to remove the container? or
does navbar mean to have container inside?

thanks in advance

**Comment by shinokada** [comment]
[NavContainer](https://github.com/themesberg/flowbite-svelte/blob/main/src/lib/navbar/NavContainer.svelte) is just a wrapper where you can use the `fluid` and class props.

```
<script lang="ts">
  import type { HTMLAttributes } from 'svelte/elements';
  import { twMerge } from 'tailwind-merge';

  interface $$Props extends HTMLAttributes<HTMLDivElement> {
    fluid?: boolean;
  }

  export let fluid: boolean = false;
</script>

<div class={twMerge('mx-auto flex flex-wrap justify-between items-center ', fluid ? 'w-full' : 'container', $$props.class)}>
  <slot />
</div>
```
